variants/icarus/platformio.ini

```diff
@@ -0,0 +1,19 @@
+[env:icarus]
+extends = esp32s3_base
+board = icarus
+board_level = extra
+board_check = true
+board_build.mcu = esp32s3
+upload_protocol = esptool
+upload_speed = 921600
+platform_packages = framework-arduinoespressif32@https://github.com/PowerFeather/powerfeather-meshtastic-arduino-lib/releases/download/2.0.16b/esp32-2.0.16.zip
```

Comment:
Is there something special about that framework? just curious.

Reply:
hi, yes it adds improved power management for esp32s